core/src/main/java/io/grpc/internal/ManagedClientTransport.java

```diff
@@ -110,6 +110,8 @@ interface Listener {
      * Called just before {@link #transportReady} to allow direct modification of transport
      * Attributes.
      */
-    Attributes filterTransport(Attributes attributes);
+    default Attributes filterTransport(Attributes attributes) {
```

Comment:
I was kinda on a fence about adding a default implementation to a pure interface, but after talking IRL I see how it makes sense. It's an optional filter, so I think we're all good here.